core/jazz_ui/src/app/pages/service-overview/service-overview.component.ts

```diff
@@ -443,8 +431,14 @@ export class ServiceOverviewComponent implements OnInit {
   }
 
   onCancelClick() {
+    if(this.service.eventScheduleEnable === false){
+      this.rateExpression.type = "none"
+    } else {
+      this.rateExpression.type = "cron"
```

Comment:
try not to hard code 'this.rateExpression.type' value. assign based on the radio button selected.

Reply:
Assigned the rateExpression type value according to the service value